src/UI/TransactionsToastList/index.tsx

```diff
@@ -33,49 +54,41 @@ function TransactionsToastList({
   const signedTransactionsToRender =
     signedTransactions || signedTransactionsFromStore;
 
-  const generatedClasses = getGeneratedClasses(
-    className,
-    shouldRenderDefaultCss,
-    {
-      wrapper:
-        'toast-messages d-flex flex-column align-items-center justify-content-sm-end',
-      toast: ''
-    }
-  );
+  const handleDeleteCustomToast = (toastId: string) => {
+    removeToast(toastId);
+    setToastsIds((toastsIds: ToastsType[]): ToastsType[] =>
```

Comment:
Maybe we could do this filtering inside of removeToast function and pass a callback to be called with the updated list?

removeToast(toastId, setToastsIds)